surveyor.py

```diff
@@ -18,6 +19,11 @@
 # list of all the different products we support
 @click.option("--threathunter", 'product', help="Use this to use Cb ThreatHunter.", flag_value="cbth", default=False)
 @click.option("--response", 'product', help="Use this to use Cb Response.", flag_value="cbr", default=True)
+@click.option("--defender", 'product', help="Use this to query Microsoft Defender for Endpoints", flag_value="defender", default=False)
+@click.option("--atp", 'product', help="Use this to query Microsoft Defender for Endpoints", flag_value="defender", default=False)
+
+@click.option("--atpcreds", 'creds', help="Use this to define the path of the ini file with your ATP credentials", type=click.Path(exists=True))
```

Comment:
I would suggest an additional option for Defender credentials, for those who are not aware of the old ATP naming scheme. Maybe `microsoftcreds` or something?

Reply:
So a doubling up like ```--atp``` and ```--defender``` or just one that's more generic such as ```--creds```?

Reply:
Probably something more generic like just `--creds` would make the most sense. Presuming you can't sensibly specify multiple credentials files for multiple EDR platforms in the same run.